src/libData/AccountData/Account.h

```diff
@@ -57,6 +57,8 @@ class Account : public Serializable
 public:
     Account();
 
+    //tells if the account is a contract accoun
+    bool isContract() const { return m_codeHash != h256(); }
```

Comment:
Can remove this function as its already merged

Reply:
Fixed.